install/error-handling.sh

```diff
@@ -188,6 +189,10 @@ cleanup() {
             --arg lineno "$lineno" \
             '{"filename": $filename, "function": $function, "lineno": $lineno|tonumber}'
         )
+        if [[ $i -eq 1 ]]; then
```

Comment:
if we're in the stacktrace of the file we failed on, we can add a context line with the command run that failed